### DISCUSSION TITLE
Error: P2032

### DISCUSSION THREAD

**Discussion by NoelHuibers** [prisma/prisma#17146] [Q&A]
Hey,

I got the Error P2032 on a project with a SQLite Database using autoincrement() on my id.
I wanted to look up the Documentation, but only 2031 and 2033 are documented. Is there an existing explanation of the Error code?

**Comment by nurul3101** [comment]
Hello @NoelHuibers 👋 

Here's an [explanation](https://www.prisma.io/docs/concepts/components/prisma-client/composite-types#default-values-for-required-fields-on-composite-types) of P2032 error code, this error code was applicable till Prisma version 4.0.0
Does it help?
<img width="698" alt="Screenshot 2023-01-05 at 2 26 04 PM" src="https://user-images.githubusercontent.com/13049676/210740437-b098dcec-baed-4b44-93f2-5f9443c3ae79.png">

**Reply by NoelHuibers** [reply]
I'm using Prisma Version 4.5 and updated to 4.8 without any change. Is there an explanation for it for 4.0.0 upwards?

**Reply by janpio** [reply]
Do you have only the error code or also an error message with some more information?
What are you trying to do when you get the error message?

**Reply by NoelHuibers** [reply]
I get the message when trying to create an entry in my table using autoincrement() not giving the entry an ID myself. I use a SQLite database.
Error message i get in Prisma Studio: 
```
"Failed to Commit changes: Invalid 'prisma.test.create()" invocation: Error converting field "ID" to expected non-nullable type "Int", found incompatible value of "null".
Details:
Type: undefined
Message: Invalid 'prisma.test.create()'
invocation:
Error converting field 'id' of expected non-nullable type "Int", found incompatible vlaue of "null".

Code: P2032

Query: [object Object]
```

I hope this helps. Let me know if you need some extra information.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
